sound/soc/sof/ipc4-topology.h

```diff
@@ -31,6 +31,12 @@
 #define SOF_IPC4_NODE_INDEX(x)	((x) & SOF_IPC4_NODE_INDEX_MASK)
 #define SOF_IPC4_NODE_TYPE(x)  ((x) << 8)
 
+/* Node ID for SSP type DAI copiers */
+#define SOF_IPC4_NODE_INDEX_INTEL_SSP(x) (((x) & 0xf) << 4)
```

Comment:
bit 7 ~ 4 is for instance id like our dai id. Can we define a macro #define xxx_INSTANCE_OFFSET 4 (choose what you like, I am not good at naming).

Reply:
I would not bother with another define which is only used in this macro.

Reply:
for bit 0 ~ 3, we don't use it now. it is for virtual slot. Maybe it will be used in future. We can add macro at that time